### ISSUE TITLE
Failure on multibyte UTF-8 characters

### ISSUE THREAD

**Issue by 2colours** [titsuki/raku-Terminal-Getpass#15]
Hello,
I installed the module but I'm damned to have non ascii characters in my password so the following happened:
```
Password: Malformed termination of UTF-8 string
in sub unix-getpass at <path> (Terminal::Getpass) line 27
```
Indeed - if you only read one byte of an UTF-8 character, you might try to decode something that isn't even valid - and even if it were, nothing good could follow from adding it to the output.

I think this is easy to fix: the processing should either be more byte-like or more character-like.

**Comment by 2colours** [comment]
https://github.com/titsuki/raku-Terminal-Getpass/pull/16 addresses this issue

**Comment by titsuki** [comment]
Fix by #16
